.github/workflows/docsbuild-release.yaml

```diff
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: 'Release Version. Example: 11.1.0'
+        required: true
+      ansible-major-version:
+        description: 'Example 11'
+        required: true
+
+env:
+  CI_COMMIT_MESSAGE: 'Add Ansible community "${{ inputs.ansible-version }}" porting guide'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CI_PR_BODY: |
+
+      ##### SUMMARY
+
+      ##### ISSUE TYPE
+
+      - Docs Pull Request
+
+      ##### COMPONENT NAME
+
+      docs/docsite/rst/porting_guides/'porting_guide_"${{ inputs.ansible-major-version }}".rst'
```

Comment:
This would need to have a placeholder that would be replaced (maybe, with `sed` too) after that step I suggested. And the post-processed value would be what you pass to `gh pr`.

Reply:
I think this has already been resolved.

Reply:
Yes, it is now. Marking as resolved.